pkg/gomason/languages/golang.go

```diff
@@ -0,0 +1,397 @@
+package languages
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/a8m/envsubst"
+	"github.com/pkg/errors"
+
+	"github.com/nikogura/gomason/pkg/gomason"
+)
+
+func init() {
+	languagesMap["golang"] = Golang{}
+}
+
+// For golang, workdir is GOPATH
+type Golang struct{}
+
+// Creates an empty but workable GOPATH in the directory specified. Returns
```

Comment:
Doc lines like these need to start with the name of the function (in this case 'CreateWorkdir') to be picked up by the godoc parser